content/docs/examples/userauthentication.md

```diff
@@ -34,22 +58,22 @@ To create the `API` and `Serverless Function` run
 cdev deploy
 ```
 
-You will get output that looks like 
+You will get an output that looks like: 
 ```
 Base API URL -> https://<your-endpoint>/live
 Routes -> FrozenDict({'/demo GET': 'ayhuxeb'})
 ```
 
-You can check that this `API` is working correctly by using the Curl command
+You can check that this `API` is working correctly by using the `Curl` command
 ```bash
 curl https://<your_endpoint>/live/demo
 ```
 
-Or by going to the url `https://<your_endpoint>/live/demo` in your browser.
+Or, by going to the url `https://<your_endpoint>/live/demo` in your browser.
 
 {{<break 1>}}
 ## Setting up Auth0
-You can create a [trail Auth0 account](https://auth0.com/signup) that has a free tier of 7,000 active users and unlimited logins with no required credit card. 
+You can create a [trial Auth0 account](https://auth0.com/signup) that has a free tier of 7,000 active users and unlimited logins with no required credit card. 
```

Comment:
Can you add a sentence about what Auth0 is? You can just modify something from their website. This will help users that have never heard of Auth0 before.